cmd/modern/root/install/mssql-base_test.go

```diff
@@ -0,0 +1,32 @@
+package install
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetDbNameIfExists(t *testing.T) {
+
+	type test struct {
+		input          string
+		expectedOutput string
+	}
+
+	tests := []test{
+		// Positive Testcases
+		{"https://example.com,myDbName", "myDbName"},
+		{"https://example.com,myDb Name", "myDb Name"},
+		{"https://example.com,myDb Na,me", "myDb Na,me"},
+		{"https://example.com,[myDb Na,me]", "[myDb Na,me]]"},
```

Comment:
>"[myDb Na,me]]"}, [](http://example.com/codeflow?start=39&length=17)

if this function is returning the string that's used for inserting into the TSQL, rename the function `getDbNameForScript` to make it clear that's not the actual name.